gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java

```diff
@@ -77,7 +77,9 @@ public final class GrpcCallContext implements ApiCallContext {
   private static final GrpcStatusCode UNAUTHENTICATED_STATUS_CODE =
       GrpcStatusCode.of(Status.Code.UNAUTHENTICATED);
 
-  static final CallOptions.Key<ApiTracer> TRACER_KEY = CallOptions.Key.create("gax.tracer");
+  // This field is made public for handwritten libraries to easily access the tracer from
+  // CallOptions
+  public static final CallOptions.Key<ApiTracer> TRACER_KEY = CallOptions.Key.create("gax.tracer");
```

Comment:
qq, having a `TRACER_KEY` in the CallOptions this a gRPC specific thing? Would it make sense to also have this for HttpJson and if so, would it make sense to make this as transport neutral as possible?

Reply:
Yes, it makes sense. Currently it is only used by [GrpcChannelUUIDInterceptor](https://github.com/googleapis/sdk-platform-java/blob/main/gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcChannelUUIDInterceptor.java#L47), if we have a need to use it in [HttpJsonHeaderInterceptor](https://github.com/googleapis/sdk-platform-java/blob/main/gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpJsonHeaderInterceptor.java#L52), then we can add it to HttpJsonCallOptions. We can refactor it when we have the need in the future.